bin/post_compile.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+# Tasks run by the Heroku Python buildpack after the compile step.
+
+# The post_compile script is run in a sub-shell, so we need to source the
+# buildpack's utils script again, so we can use set-env/set-default-env:
+# https://github.com/heroku/heroku-buildpack-python/blob/master/bin/utils
+source $BIN_DIR/utils
+
+# Override the hostname that is displayed in New Relic, so the Dyno name
+# (eg "web.1") is shown, rather than "Dynamic Hostname". $DYNO is quoted
+# so that it's expanded at runtime on each dyno, rather than now.
+set-env NEW_RELIC_PROCESS_HOST_DISPLAY_NAME '$DYNO'
```

Comment:
I think this was got deleted as part of https://github.com/mozilla-services/tecken/commit/b6c427fa4e94a26d796f93ca4ea545f37e02cadd because it turned up when doing `grep`. I don't think this file gets used anywhere.

Reply:
I don't see it anymore either, and clearly we're not running on Heroku.